### PR TITLE
Create testfile

### PR DESCRIPTION
**Pull request by ZainabSayyed**
#add new branch , commit it and pull request for discussion.

### CODE REVIEW

**Comment by ZainabSayyed** [comment]
Add Myself to the Map #926

File: testfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#Hi this is just a test file.
+githubHandle: Z Sayyed
+latitude: 37.5483° N
+longitude: 121.9886° W
```
Comment (hectorsector) [inline testfile]: @ZainabSayyed great start, but try replacing the contents of this file with:
```
githubHandle: zainabsayyed
latitude: 37.5483
longitude: 121.9886
```

Then, rename the file `_pins/zainabsayyed.json`.

File: SalvadorRuiz.json [hunk 2]
```diff
@@ -3,3 +3,9 @@ githubHandle: SalvadorRuizIII
 latitude: 28°32′18″ 
 longitude: 81°22′45″ 
 ---
+
+---
+githubHandle: Zsayyed
+latitude: 37.5483° N 
+longitude: 121.9886° W
+---
```
Comment (hectorsector) [inline SalvadorRuiz.json]: @ZainabSayyed this file belongs to @SalvadorRuizIII please remove your content and add it to your own file.

**Comment by ZainabSayyed** [comment]
*@hectorsector*  I remember changes I did but unable to find it. Can you
guide me how can I search specific ?

Thanks
@zainabsayyed

File: testfile (hunk 1 above)
Comment (ZainabSayyed) [inline testfile]: @hectorsector I accommodate changes. I would though prefer to ignore or undo changes.

**Comment by ZainabSayyed** [comment]
Hi;
Can anyone let me know where can I find webinar held on 27 Sep and 28 Sep this week. I like to practice and refer it!

**Pull request merged**